Template.ProcessCube.Project/apps/Template.ProcessCube.Project/Template.ProcessCube.Project.csproj

```diff
@@ -28,6 +24,8 @@
     <None Include="$(SpaRoot)**" Exclude="$(SpaRoot)node_modules\**" />
   </ItemGroup>
 
+  <Import Project="$(MSBuildProjectName).targets" Condition="Exists('$(MSBuildProjectName).targets')" />
+  <Import Project="$(MSBuildProjectName).ui-tests.targets" Condition="Exists('$(MSBuildProjectName).ui-tests.targets')" />

```

Comment:
Es gibt von der Template Engine aus auch eine bestimmte Syntax wie du Code-Blöcke bedingt in- oder excluden kannst. Finde das aber auch ok, könnte aber zu Inkonsistenz führen, wenn an anderen Stellen, die Syntax der Template Engine benutzt wird

Reply:
Das mit dem inkludieren über die Template CodeBlöcke könnte ich zusätzlich einbauen, damit nicht irgendwann ganz viele unnötige Zeilen vorhanden sind.
Das Anlegen einer eigenen Targets-Datei finde ich sinnvoll, weil dann alles von den Tests wirklich an einer Stelle abgelegt ist und die normale .targets dann nicht zu voll wird.
Mit den Inkonsitenzen müsstest du mir nochmal erklären.